Classes/UIAutomationHelper.m

```diff
@@ -177,7 +177,7 @@ - (void)deactivateAppForDuration:(NSNumber *)duration {
         NSAssert([[[[self target] frontMostApp] name] isEqual:@"SpringBoard"], @"If reactivation is failing, the app is likely still open to SpringBoard.");
         
         // Tap slightly above the middle of the screen, otherwise it doesn't resume on an iPad Pro
-        [[[self target] frontMostApp] tapWithOptions:@{@"tapOffset": @{@"x": @(.5), @"y": @(.36)}}];
+        [[[self target] frontMostApp] tapWithOptions:@{@"tapOffset": @{@"x": @(.7), @"y": @(.36)}}];
```

Comment:
Can we derive any of this programmatically?

Reply:
So i looked into this for a few hours this weekend and i think it's a possibility but i'll need to dig into it more:

```
lldb) po [[[[self target] elements] firstObject] valueForKey:@"logInfoWithChildren"]
{
    className = UIAApplication;
    convertedRect = "{{0, 0}, {1242, 2688}}";
    elements =     (
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            elements =             (
                                {
                    className = UIAImage;
                    convertedRect = "{{-31.5, 217.00000000000003}, {24, 24}}";
                    name = IconNewDot;
                    rect = "{{-10.5, 72.333333333333343}, {8, 8}}";
                },
                                {
                    className = UIAImage;
                    convertedRect = "{{-31.5, 217.00000000000003}, {24, 24}}";
                    name = IconNewDot;
                    rect = "{{-10.5, 72.333333333333343}, {8, 8}}";
                },
                                {
                    className = UIAImage;
                    convertedRect = "{{0, 0}, {1242, 2688}}";
                    rect = "{{0, 0}, {414, 896}}";
                }
            );
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            elements =             (
                                {
                    className = UIAScrollView;
                    convertedRect = "{{0, 0}, {1242, 2688}}";
                    rect = "{{0, 0}, {414, 896}}";
                },
                                {
                    className = UIAElement;
                    convertedRect = "{{0, 0}, {1242, 2688}}";
                    elements =                     (
                                                {
                            className = UIAElement;
                            convertedRect = "{{195.6585027284334, 396.45415083255159}, {850.68299454313319, 1841.0916983348968}}";
                            name = "com.square.Test-Host";
                            rect = "{{65.219500909477802, 132.15138361085053}, {283.5609981810444, 613.69723277829894}}";
                            value = Active;
                        }
                    );
                    name = AppSwitcherContentView;
                    rect = "{{0, 0}, {414, 896}}";
                },
                                {
                    className = UIAButton;
                    convertedRect = "{{63, 2505}, {1116, 120}}";
                    name = "";
                    rect = "{{21, 835}, {372, 40}}";
                }
            );
            name = SBSwitcherWindow;
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            rect = "{{0, 0}, {414, 896}}";
        },
                {
            className = UIAWindow;
            convertedRect = "{{0, 0}, {1242, 2688}}";
            rect = "{{0, 0}, {414, 896}}";
        }
    );
    name = SpringBoard;
    rect = "{{0, 0}, {414, 896}}";
}
```

It's just accessing that test host element because everytime i try to get `elements` from ` UIAElement` it comes up as nil but not sure how to get anything below that. Need to work more with uiautomation

Reply:
Added an issue #1178